src/installing-workflow/configuring-object-storage.md

```diff
@@ -51,6 +51,10 @@ The builder reads credentials from the below locations on the filesystem.
 - Key: `/var/run/secrets/object/store/access-key-id`
 - Secret `/var/run/secrets/object/store/access-key-secret`
 
+### Helm Chart
+
+If you are using the [Helm Chart for Workflow][helm-chart], put your credentials in the [`minio-user` secret][minio-user-secret] (under `access-key-id` and `access-secret-key`) before you `helm install`. For more information, see the [installation instructions][helm-install] for more details on using Helm.
```

Comment:
Added the notes about base64 encoding in bc58dc4e67ae9b110a1be96833c26a71489e10c3